numpy_financial/_financial.py

```diff
@@ -26,6 +26,24 @@
                 'start': 1,
                 'finish': 0}
 
+# Define custom Exceptions
+
+class NoRealSolutionException(Exception):
+    """ No real solution to the problem. """
+    def __init__(self, message):
+        self.message = message
+
+    def __str__(self):
+        return self.message
```

Comment:
I *think*

```suggestion
class NoRealSolutionException(Exception):
    """No real solution to the problem."""
   pass
```

is enough. At least it is for newer Python versions. Could you check if this is needed?

Reply:
Indeed, you can omit all those lines inside the class, since it is the default configuration (as I had coded it initially). However, I preferred to code it explicitly in case future development is wanted to be done to the exceptions, such as for instance logging. But if you do not envision this, you can definitely code it in the short way!

Reply:
I'd prefer it to be concise. If, in the future, we want to edit it then we can.